src/Swift.Bindings/tests/IntegrationTests/FunctionalTests/Protocols/ProtocolsTests.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Swift.ProtocolsTests;
+using Xunit;
+
+namespace BindingsGeneration.FunctionalTests
+{
+    public class ProtocolsTests : IClassFixture<ProtocolsTests.TestFixture>
+    {
+        private readonly TestFixture _fixture;
+
+        public ProtocolsTests(TestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        public class TestFixture
+        {
+            static TestFixture()
+            {
+                InitializeResources();
+            }
+
+            private static void InitializeResources()
+            {
+                // Initialize
+            }
+        }
+
+        [Fact]
+        public void ProtocolIsProjected()
+        {
+            // This test is to ensure that the protocol is projected, it just needs to compile
+            Assert.NotNull(typeof(ISwiftPrintable));
```

Comment:
I'm unsure how this test works. Could you explain how compiling the source code means that we verified that `ISwiftPrintable` is projected?

Reply:
Should be a bit more readable now. The idea is that we currently project protocols as empty interfaces so there is no much to test except the fact that it exists and it is an interface.